lib/remote_record/reference.rb

```diff
@@ -29,7 +29,10 @@ def remote_record_config
     included do
       attr_accessor :fetching
 
+      scope :skip_fetching, -> { where(fetching: false) }
```

Comment:
I'm not quite sure I understand why a reference would have a column called `fetching`

Reply:
And also unclear to me why this would involve a `scope`

Reply:
`fetching` isn't a column, it's a transient attribute - it's stored on the instance, but not in the database.

The scope is necessary so that records can be instantiated from an `ActiveRecord::Relation` with the transient attribute set.